src/main/java/darren/assistant/tasks/DeadlinesTask.java

```diff
@@ -0,0 +1,63 @@
+package darren.assistant.tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a task with a deadline.
+ * <p>
+ * A {@code DeadlinesTask} has a description and a specific due date/time
+ * by which the task must be completed.
+ * </p>
+ */
+public class DeadlinesTask extends Task {
```

Comment:
Perhaps singular should be used for the class name?
```suggestion
public class DeadlineTask extends Task {
```